base_classes/NXtransformations.nxdl.xml

```diff
@@ -108,6 +126,143 @@
+		Three point detectors are positioned in this reference:
+
+		* *transmission*:
+			* point detector in the beam
+			* 20 cm downstream from the sample (the origin of the reference frame)
+		* *vertical*:
+			* point detector 10 cm downstream from the sample
+			* making an angle of 5 degrees with the beam w.r.t. the sample
+			* positioned in the XZ-plane above the XY-plane
+		* *horizontal*:
+			* point detector 11 cm downstream from the sample
+			* making an angle of 6 degrees with the beam w.r.t. the sample
+			* positioned in the XY-plane above the XZ-plane
+
+		The coordinates of the point detectors in the laboratory reference frame are
+
+		* *transmission*: :math:`X_\text{lab} = T_x(20) . X_d`
+		* *vertical*: :math:`X_\text{lab} = R_y(-5) . T_x(10) . X_d`
+		* *horizontal*: :math:`X_\text{lab} = R_x(-90) . R_y(-6) . T_x(11) . X_d`
+
+		where
+
+		* :math:`T_x`, :math:`T_y`, :math:`T_z`: active transformation matrices for translation along the X, Y and Z axes
+		* :math:`R_x`, :math:`R_y`, :math:`R_z`: active transformation matrices for rotation around the X, Y and Z axes
+		* :math:`X_d` is a coordinate in the detector reference frame.
+
+		Note that as these are point detectors, we only have one coordinate :math:`X_d=[0,0,0,1]^T`.
+
+		.. code-block::
+
+		  entry:NXentry
+		    instrument:NXinstrument
+		      vertical:NXdetector
+		        depends_on=position/distance
+		        position:NXtransformations
+		          distance=10            # move downstream from the sample
+		            @depends_on=polar
+		            @units=cm
+		            @vector=[1 0 0]
+		          polar=5                # title above the horizontal plane
+		            @depends_on=azimuth
+		            @units=degrees
+		            @vector=[0 -1 0]
+		          azimuth=0              # stay in the vertical plane
+		            @depends_on=/entry/coordinate_system/beam
+		            @units=degrees
+		            @vector=[-1 0 0]
+		      horizontal:NXdetector
+		        depends_on=position/distance
+		        position:NXtransformations
+		          distance=11            # move downstream from the sample
+		            @depends_on=polar
+		            @units=cm
+		            @vector=[1 0 0]
+		          polar=6                # title above the horizontal plane
+		            @depends_on=azimuth
+		            @units=degrees
+		            @vector=[0 -1 0]
+		          azimuth=90             # rotate to the horizontal plane
+		            @depends_on=/entry/coordinate_system/beam
+		            @units=degrees
+		            @vector=[-1 0 0]
+		      transmission:NXdetector
+		        depends_on=position/distance
+		        position:NXtransformations
+		          distance=20            # move downstream from the sample
+		            @depends_on=/entry/coordinate_system/beam
+		            @units=cm
+		            @vector=[1 0 0]
+		    coordinate_system:NXtransformations
```

Comment:
why would beam depends_on gravity why do I think NXcoordinate_system is cleaner/more explicit?